File: Programmers/월간코드챌린지/Lv1음양더하기.java
public class Lv1음양더하기 {

    public int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;
        for (int i = 0; i < absolutes.length; i++) {
            int number = absolutes[i];
            if (!signs[i]) {
                number *= -1;
            }
            answer += number;
        }
        return answer;
    }

}

File: Programmers/월간코드챌린지/Lv1음양더하기.kt
class Solution {
    fun solution(absolutes: IntArray, signs: BooleanArray): Int {
        var answer = 0
        for (i in absolutes.indices) {
            answer += getNumber(signs[i], absolutes[i])
        }
        return answer
    }

    private fun getNumber(sign: Boolean, number: Int): Int {
        if (sign) {
            return number
        }
        return -number
    }
}